sql-plugin/src/main/scala/com/nvidia/spark/rapids/literals.scala

```diff
@@ -622,7 +622,7 @@ case class GpuLiteral (value: Any, dataType: DataType) extends GpuLeafExpression
 
   override def toString: String = value match {
     case null => "null"
-    case binary: Array[Byte] => s"0x" + DatatypeConverter.printHexBinary(binary)
+    case binary: Array[Byte] => s"0x" + ApacheHex.encodeHex(binary, false)
```

Comment:
This needs to use `encodeHexString`, otherwise we're calling toString on Array[Byte] which isn't what we want.
```suggestion
    case binary: Array[Byte] => s"0x${ApacheHex.encodeHexString(binary, false)}"
```

Reply:
thanks, autocomplete typo, too bad premerge  did not catch it

Reply:
will have to do `ApacheHex.encodeHex(binary, false)` because there is no `Hex.encodeHexString(binary, false)` in earlier versions of commons-codec before 1.11 used in Spark before 3.2 https://issues.apache.org/jira/browse/SPARK-33762